Project.toml

```diff
@@ -6,11 +6,13 @@ desc = "A high level API for GDAL - Geospatial Data Abstraction Library"
 version = "0.6.0"
 
 [deps]
+ColorTypes = "3da002f7-5984-5a60-b8a6-cbb66c0b333f"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 DiskArrays = "3c3547ce-8d99-4f5e-a174-61eb10b00ae3"
 GDAL = "add2ef01-049f-52c4-9ee2-e494f65e021a"
 GeoFormatTypes = "68eda718-8dee-11e9-39e7-89f7f65f511f"
 GeoInterface = "cf35fbd7-0cd7-5166-be24-54bfbe79505f"
+ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
```

Comment:
This needs a compat entry.

Reply:
Thanks! Added in https://github.com/yeesian/ArchGDAL.jl/pull/179/commits/fcaba40d908d2778799fb3643580196f88805365. Feel free to ping me if there are any issues with it.